index.html

```diff
@@ -270,7 +270,28 @@ <h5>LET'S CONNECT</h5>
           <li class="skills-text">Skills</li>
             <img class="disabled-menu2" src="./imgs/disabled-menu/disabled-menu.svg" alt="disabled menu logo" />
       </ul>
+      </section>
+      <!-- Contact Form Section -->
+      <section id="section-contact-form" class="contact-form">
+        <form id="form" class="cform" action="https://formspree.io/f/mrgvnevk">
+          <h2>Contact me</h2>
+          <p id="contact-form-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+            <br> 
+            I’d love to help with it!
+            <br>
+          </p>
+          <div class="form-group">
+            <input type="text" id="name" name="name" placeholder="Lorem Ipsum" maxlength="30" required />
+          </div>
+          <div class="form-group">
+            <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required />
+          </div>
 
+          <div class="form-group">
+            <textarea id="message" name="message" placeholder="Write your message here..." maxlenght="500" required /></textarea>
+          </div>
+          <input id="submit" type="submit" value="Get in touch">
```

Comment:
- [ ] For good user experience, Please ensure that you implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

![image](https://github.com/NoelLincoln/mportfolio/assets/63473584/c705b2c2-eee2-46b4-bcb4-f19750469f76)

Reply:
It's done! ✍️✅